inc/osvr/Common/SerializationTraits.h

```diff
@@ -510,6 +511,15 @@ namespace common {
                 f(val.data[2]);
             }
         };
+
+        template <>
+        struct SimpleStructSerialization<StringID>
+            : SimpleStructSerializationBase {
+            template <typename F, typename T> static void apply(F &f, T &val) {
+                f(val.m_val);
```

Comment:
kind of looks like you're accessing a private member: I'd suggest having a member function named `value()` or similar that returns a reference.